src/helpers/__tests__/insights-test.ts

```diff
@@ -0,0 +1,164 @@
+import insights, {
+  writeDataAttributes,
+  readDataAttributes,
+  hasDataAttributes,
+} from '../insights';
+
+const makeDomElement = (html: string): HTMLElement => {
+  const div = document.createElement('div');
+  div.innerHTML = html;
+  return <HTMLElement>div.firstElementChild || div;
```

Comment:
```suggestion
  return div.firstElementChild as HTMLElement || div;
```

See: https://basarat.gitbooks.io/typescript/docs/types/type-assertion.html#as-foo-vs-foo.

Reply:
thanks!